source/sass/buyers-guide/views/catalog.scss

```diff
@@ -349,10 +351,67 @@ body.catalog {
       height: var(--sticky-height);
     }
 
-    &.search-active .current-creepiness {
+    &.search-active .creep-o-meter-information {
       position: fixed;
       right: 2em;
       bottom: 2em;
+      top: initial;
+    }
+
+    @media (max-width: $bp-md) {
+      &.search-active {
+        height: var(--medium-sticky-height);
+      }
+    }
+
+    @media (max-width: $bp-sm) {
+      &.search-active {
+        height: var(--small-sticky-height);
+      }
+    }
```

Comment:
We wanna write mobile first style code where possible so rather than using `max-width`, let's covert these to `min-width`

```scss
    &.search-active {
      height: var(--small-sticky-height);

      @media (min-width: $bp-md) {
        height: var(--medium-sticky-height);
      }

      @media (min-width: $bp-lg) {
        height: ...;
      }
    }
```

Reply:
fun fact: this should not be in this PR - that's code I copied from #5557 >_>;;

Reply:
alrighty!